### PR TITLE
(#3041) Fixed the locations of weights_only argument in the tutorial file

### LINKED ISSUE #3041: [BUG] - There are 2 incorrect arguments positioning in "tutorials\beginner_source\saving_loading_models.py"
### Add Link

https://github.com/pytorch/tutorials/blob/main/beginner_source/saving_loading_models.py

### Describe the bug

"weights_only" is augument of torch.load, not for model.load_state_dict
Issue at line 156 and 410.

Here is the doc:
https://pytorch.org/docs/stable/generated/torch.load.html

### Describe your environment

Windows 11, PyTorch 2.4.1

### PR DESCRIPTION
**Pull request by jaewoosong**
Fixes #3041 

## Description
Fixed the locations of `weights_only` argument in the tutorial file.

## Checklist

- [X] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [X] Only one issue is addressed in this pull request
- [X] Labels from the issue that this PR is fixing are added to this pull request
- [X] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Comment by jaewoosong** [comment]
Since the 'bug' label was added to issue https://github.com/pytorch/tutorials/issues/3041 I tried to add the 'bug' label here, but it seems I do not have permission to add the label by myself. 😢

**Comment by jaewoosong** [comment]
Oh, it seems the same patch is applied to the main branch!
